html/semantics/embedded-content/the-iframe-element/iframe-loading-lazy-history-replaceState.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<title>History state change for iframe loading='lazy' before it is loaded: history.replaceState</title>
+<iframe data-src="about:blank#foo" src="support/blank.htm?src" loading="lazy" onload="window.gotIframeLoadEvent = true;"></iframe>
```

Comment:
Similar to https://github.com/web-platform-tests/wpt/pull/45650#discussion_r1578525052, maybe this can be `about:blank#replace`?